src/components/common/NewChildCard.js

```diff
@@ -26,18 +26,20 @@ function NewChildCard(props) {
       <div className="Players">
         <div className="playheading">
           <h2>Players</h2>
-        </div>
-        <div className="childCardButtons">
-          <button className="addPlayerButton" onClick={addPlayerPush}>
-            <PlusCircleOutlined /> Add Player
-          </button>
-          {props.props.parent.children ? (
-            <button className="editPlayerButton" onClick={editPlayerPush}>
-              <EditOutlined /> Edit Players
-            </button>
-          ) : null}
+
+          <div className="childdivns">
+            <Button className="addPlayerButton" onClick={addPlayerPush}>
+              <PlusCircleOutlined /> Add Player
+            </Button>
+            {props.props.parent.children ? (
```

Comment:
If it works it works, but props.props seems weird to read.  Would it make more sense to destructure to just parent then... parent.children on line 34?